netzob/src/netzob/Import/PCAPImporter/PCAPImporter.py

```diff
@@ -87,6 +88,11 @@ class PCAPImporter(object):
     b'RESdecrypt#\\x00\\x00\\x00\\x00\\x06\\x00\\x00\\x00abcdef'
     b'CMDbye#\\x00\\x00\\x00\\x00'
     b'RESbye#\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00'
+    
+    PCAP Files with unsupported Layers on OSI Layer 2 can be imported as RawMessages if the importLayer is 1
+    >>> messages = PCAPImporter.readFile("./test/resources/pcaps/atm_capture1.pcap", importLayer=1).values()
+    >>> print(repr(messages[0].data))
+    b'E\x00\x00T\x17\x82\x00\x00@\x01U\xd3\xc0\xa8F\x01\xc0\xa8F\x02\x08\x00\xfa`Of\x00\x009\xd9\x121\x00\x08w#\x08\t\n\x0b\x0c\r\x0e\x0f\x10\x11\x12\x13\x14\x15\x16\x17\x18\x19\x1a\x1b\x1c\x1d\x1e\x1f !"#$%&\'()*+,-./01234567'
```

Comment:
This docstring line produces an error in Travis CI when parsing doctests. Could you replace the initial doctring opener (""") with (r""") to avoid docstring parsing error. I think this will resolve the issue.

Reply:
I changed it to the r"""" docstring, but i don't use Travis CI, so I'm not sure if it solved the problem.